extensions/dependencies/guppy.html

```diff
@@ -8,3 +8,27 @@
     '/third_party/static/guppy-be6f14/src/transform.xsl',
     '/third_party/static/guppy-be6f14/src/symbols.json');
 </script>
+
+<style>
+    .guppy_active {
+      box-shadow: 0 0 5px rgba(81, 203, 238, 1) !important;
```

Comment:
In that case, why not pull the inline style elements into their own class? You have control over them: https://github.com/oppia/oppia/blob/develop/extensions/interactions/MathExpressionInput/MathExpressionInput.html